srcalc/src/test/java/gov/va/med/srcalc/vista/RpcVistaPatientDaoTest.java

```diff
@@ -370,4 +386,44 @@ public final void testInvalidDnrNotes() throws Exception
         // This would fail to retrieve DNR notes and the notes would be empty.
         assertEquals(0, patient.getDnrNotes().size());
     }
+    
+    @Test
+    public final void testFullVitals() throws Exception
+    {
+        final VistaProcedureCaller caller = mockVistaProcedureCaller();
+        when(caller.doRpc(
+                RADIOLOGIST_DUZ, RemoteProcedure.GMV_LATEST_VM, String.valueOf(PATIENT_DFN)))
+            .thenReturn(FULL_VITALS);
+        final RpcVistaPatientDao dao = new RpcVistaPatientDao(caller, RADIOLOGIST_DUZ);
+        final Patient patient = dao.getPatient(PATIENT_DFN);
+        final ImmutableList<Double> expectedValues = ImmutableList.of(178.0, 71.0, 24.88);
```

Comment:
Oh this is just unfair! You've said multiple times to put things into a list to check for equality. :(